src/editor/app.js

```diff
@@ -390,6 +390,26 @@ UiDriver.registerEventHandler("C_CMD_SET_THEME", function(msg, data, prevReturn)
     editor.setOption("theme", data.name);
 });
 
+UiDriver.registerEventHandler("C_CMD_SET_FONT", function(msg, data, prevReturn) {
+
+    var fontSize = (data.size != "" && data.size>0) ? ("font-size:" + data.size + "px;") : "";
+
+    var fontFamily = data.family != "" ? ("font-family:'" + data.family + "';") : "";
+
+    var styleTag = document.getElementById('userFont');
+
+    if( styleTag  ){ 
+        styleTag.innerHTML = ".editor > .CodeMirror { " + fontFamily + fontSize + " }";
+        styleTag.appendTo('head');
```

Comment:
@JuBan1 Pseudocode:

    for (var sheet of document.styleSheets) {
        if (sheet.ownerNode && sheet.ownerNode.id === 'userFont') {
            sheet.cssRules[0].style.fontFamily = something;
            sheet.cssRules[0].style.fontSize = something
        }
    }

I feel like this is more efficient, but I didn't test it. It all depends if it's more efficient to loop through DOM nodes or to re-parse CSS code.